CHANGELOG.md

```diff
@@ -49,6 +49,10 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidmm` (NI-DMM)
     * #### Added
+        * API parity with next NI-DMM release.
+            * Methods added:
+                * `convert_absolute_to_digits`
+                * `convert_digits_to_absolute`
```

Comment:
these functions are ancient and were left out of `nidmm` on purpose

Reply:
I guess we'll have to mark the grpc and python derivations as codegen `no` correct? I glanced at the history but it doesn't seem to have changed. So I'm curious what process caused them to be omitted before.

Reply:
They were added in [AB!378345](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/378345?_a=files&path=/src/nidmm/nidmmBaseDistribution/nidmmAPIMetadata/source/nidmmAPIMetadata) because it was apparently needed for another use case, but they neglected to add 'codegen_method':'no' for gRPC/Python like is done for every other function in that file.

Reply:
Removed